cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -126,6 +126,7 @@ static gpi_objtype_t to_gpi_objtype(int32_t vpitype) {
 
         case vpiStructVar:
         case vpiStructNet:
+        case vpiUnionNet:
```

Comment:
Real nitpick, sorry: the order of the existing items is "Var, Net", please do it also like that (also in line 222).